extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -361,7 +361,10 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
 
   /** @override */
   extractSize(responseHeaders) {
-    setGoogleLifecycleVarsFromHeaders(responseHeaders, this.lifecycleReporter_);
+    if (this.lifecycleReporter_) {
+      setGoogleLifecycleVarsFromHeaders(
```

Comment:
consider changing this to handle passing null for the second arg so you don't have to wrap in an if here and below